doc/source/user_guide/hpc_slurm.rst

```diff
@@ -0,0 +1,611 @@
+request to your HPC cluster administrator to have installed in all the compute
+nodes a suitable Python version.
+If this is not an option, visit :ref:`ref_ansys_provided_python`.
+
+.. _ref_ansys_provided_python:
+
+Using ANSYS provided Python installation
+----------------------------------------
+
+**For development purposes only**
+
+In certain HPC environments the possibility of installing a different Python version
+is limited for security reasons.
+
+In those cases, the Python distribution available within the ANSYS installation can be used.
+This Python distribution is a customized Python (CPython) version for ANSYS products use only, and
+its usage is **discouraged** except for very advanced users and user cases.
+
+This Python distribution is in:
+
+.. code-block:: text
+
+    /ansys_inc/v%MAPDL_VERSION%/commonfiles/CPython/3_10/linx64/Release/python
+
+whereas ``%MAPDL_VERSION%`` is the 3 digits ANSYS version. For instance for ANSYS 2024R2:
+
+.. code-block:: text
+
+    /ansys_inc/v242/commonfiles/CPython/3_10/linx64/Release/python
+
+
+From ANSYS 2024R1, the Python version included in the unified installer is CPython 3.10.
+Previous versions were including CPython 3.7 (``/commonfiles/CPython/3_7/linx64/Release/python``).
+
+Because ANSYS installation needs to be available to all the compute nodes to run simulations using them,
+this Python distribution is normally also available to the compute nodes.
+Hence, you can use it to create your own virtual environment.
+
+Due to the particularities of this Python distribution, you need to follow the following steps to create
+a virtual environment accessible to the compute nodes.
+
+1. Set Python path environment variable:
+
+   .. code-block:: console
+
+      user@machine:~$ export PY_PATH=/ansys_inc/v241/commonfiles/CPython/3_10/linx64/Release/Python
+
+2. Patch ``PATH`` and ``LD_LIBRARY_PATH`` *(Only required for ANSYS 2024R1 or older)*:
+
+   .. code-block:: console
+
+      user@machine:~$ PATH=$PY_PATH/bin:$PATH  # Patching path
+      user@machine:~$ LD_LIBRARY_PATH=$PY_PATH/lib:$LD_LIBRARY_PATH  # Patching LD_LIBRARY_PATH
+
+3. Then, on the same terminal, you can proceed to create your own virtual
+   environment and activate it:
+
+   .. code-block:: console
+
+      user@machine:~$ $PY_PATH -m venv /home/user/.venv
+      user@machine:~$ source /home/user/.venv
+
+4. Install PyMAPDL:
+
+   .. code-block:: console 
+
+      (.venv) user@machine:~$ python -m pip install ansys-mapdl-core
+
+5. Use it to launch simulations, using ``srun``:
+
+   .. code-block:: console
+
+      (.venv) user@machine:~$ srun pymapdl_script.py
+
+   or ``sbatch``:
+
+   .. code-block:: console
+
+      (.venv) user@machine:~$ sbatch job.sh
+      Submitted batch job 1
+
+
+Advanced configuration
+======================
+
+In this section, some advanced ideas are drafted for you to explore when using
+PyMAPDL on HPC clusters.
+
+Advanced Job Management
+-----------------------
+
+Job Dependencies
```

Comment:
```suggestion
Advanced job management
-----------------------

Job dependencies
```
Vale is obviously not set up in this repo or you'd be getting tons of violations!